packages/upload-client/src/blob.js

```diff
@@ -279,30 +279,10 @@ export async function add(
     /** @type {import('@ucanto/interface').SignerArchive<import('@ucanto/interface').DID, typeof ed25519.signatureCode>} */
     (nextTasks.put.task.facts[0]['keys'])
   )
-  const httpPut = HTTPCapabilities.put.invoke({
-    issuer: derivedSigner,
-    audience: derivedSigner,
-    with: derivedSigner.toDIDKey(),
-    nb: {
-      body: {
-        digest,
-        size,
-      },
-      url: {
-        'ucan/await': ['.out.ok.address.url', nextTasks.allocate.task.cid],
-      },
-      headers: {
-        'ucan/await': ['.out.ok.address.headers', nextTasks.allocate.task.cid],
-      },
-    },
-    facts: nextTasks.put.task.facts,
-    expiration: Infinity,
-  })
 
-  const httpPutDelegation = await httpPut.delegate()
```

Comment:
I removed this because it effectively recreates same task that we already have reference to.